### ISSUE TITLE
Create a "member resource" submenu under the "Members" header link

### ISSUE THREAD

**Issue by necopinus** [vgururao/yakcollective#26]
- [X] Members
- [ ] [Code of conduct](https://roamresearch.com/#/app/ArtOfGig/page/i92e8kE2x)
- [ ] [Onboarding](https://docs.google.com/document/d/1I63ZKu8o0DpBG0tWhssIsCinlXeBKO-xGyvmIMXXAWE/)
- [ ] Skills graph (plug into Vinay's "Yak Map" project)
- [ ] [Chat calendar](https://calendar.google.com/calendar/embed?src=o995m43173bpslmhh49nmrp5i4%40group.calendar.google.com)

**Comment by necopinus** [comment]
The code of conduct should definitely be linked to the "join" and "welcome" pages.

**Comment by necopinus** [comment]
For Vinay's project, see the `#yak-network-map` channel.

**Comment by tilnow** [comment]
does this require a login?

**Comment by necopinus** [comment]
I don't think so right now. Login systems get complicated, and I think we should steer away from them for now. Think of this as "resources useful for members, but okay to be public."
